src/Components/TMS/Organisms/FOTMSDashboardNavbar/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import type { FC } from "react";
+// import Router from "next/router";
+import { Box, Container } from "@mui/material";
+
+import { styled } from "@mui/material/styles";
+
+import { FOTMSSidebar } from "..";
+
+import MuiAppBar, { AppBarProps as MuiAppBarProps } from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+
+import Typography from "@mui/material/Typography";
+
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import IDashboardNavbarProps from "./DashboardNavbar.interface";
+
+const drawerWidth = 250;
+
+const Main = styled("main", { shouldForwardProp: prop => prop !== "open" })<{
+  open?: boolean;
+}>(({ theme, open }) => ({
+  transition: theme.transitions.create("margin", {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  ...(open && {
+    transition: theme.transitions.create("margin", {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    marginLeft: `${drawerWidth}px`,
+  }),
+}));
+interface AppBarProps extends MuiAppBarProps {
+  open?: boolean;
+}
+
+const AppBar = styled(MuiAppBar, {
+  shouldForwardProp: prop => prop !== "open",
+})<AppBarProps>(({ theme, open }) => ({
+  transition: theme.transitions.create(["margin", "width"], {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  ...(open && {
+    width: `calc(100% - ${drawerWidth}px)`,
+    marginLeft: `${drawerWidth}px`,
+    transition: theme.transitions.create(["margin", "width"], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  }),
+}));
+
+export const FOTMSDashboardNavbar: FC<IDashboardNavbarProps> = ({
+  testID,
+  popUp,
+  children,
+  content,
+}) => {
+  const [open, setOpen] = React.useState(false);
```

Comment:
for boolean value, please add prefi is -> isDrawerOpen

Reply:
noted